src/data/tms.ts

```diff
@@ -67954,13 +67955,13 @@ export const tmSpecies: TmSpecies = {
     SpeciesId.HISUI_LILLIGANT,
     SpeciesId.HISUI_DECIDUEYE,
   ],
-};
+});
 
 interface TmPoolTiers {
   [key: number]: ModifierTier;
 }
 
-export const tmPoolTiers: TmPoolTiers = {
+export const tmPoolTiers: Readonly<TmPoolTiers> = deepFreeze({
```

Comment:
NIT: Can be simplified by using the generic type _argument_

```suggestion
export const tmPoolTiers = deepFreeze<TmPoolTiers>({
```